src/libraries/System.Diagnostics.Process/src/Microsoft/Win32/SafeHandles/SafeProcessHandle.Unix.cs

```diff
@@ -37,6 +37,9 @@ public sealed partial class SafeProcessHandle : SafeHandleZeroOrMinusOneIsInvali
 
         internal int ProcessId { get; }
 
+        /// <summary>Closes the handle.</summary>
+        /// <returns>On Windows, returns <see langword="true" /> if the handle is closed successfully; <see langword="false" /> otherwise. On Unix, always returns <see langword="true" /></returns>
+        /// <remarks>On Windows, to get extended error information, call <see cref="System.Runtime.InteropServices.Marshal.GetLastWin32Error()" />.</remarks>
```

Comment:
I added this manually (and copied it to the Unix file). There were no docs for this API in MS Docs:  https://github.com/dotnet/dotnet-api-docs/blob/dc599c5db9e3464765b4e8c319bc22e525f4eebf/xml/Microsoft.Win32.SafeHandles/SafeProcessHandle.xml#L170-L172